src/pages/welcome/style.css

```diff
@@ -42,3 +42,8 @@
 	grid-template-columns: 1fr 1fr;
 	gap: 24px;
 }
+.welcome-form-inputs .error-message {
```

Comment:
If this error message styling is site-wide it can go into one of the global css files